core/unix/signal.c

```diff
@@ -151,8 +151,13 @@ sig_is_alarm_signal(int sig)
 #define APP_HAS_SIGSTACK(info) \
     ((info)->app_sigstack.ss_sp != NULL && (info)->app_sigstack.ss_flags != SS_DISABLE)
 
+// Under normal circumstances the app_sigaction is always fully initialized,
```

Comment:
style: C-style comments /* */ in C files

Reply:
Ah duh, didn't even notice surrounding code.